.env

```
@@ -1,2 +1,7 @@
 VITE_REACT_APP_VERSION=18.2.0
 VITE_APPLICATION_NAME=Plymouth Housing
+URL = 'https://salmon-island-01be9bf1e-staging.westus2.5.azurestaticapps.net/pick-your-name'
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)